plugins/async-node/jvm/src/test/kotlin/com/intuit/playerui/plugins/asyncnode/AsyncNodePluginTest.kt

```diff
@@ -144,7 +144,7 @@ internal class AsyncNodePluginTest : PlayerTest() {
             player.start(asyncNodeFlowSimple)
         }
         Assertions.assertTrue(count == 2)
-        Assertions.assertEquals(3, update?.getList("actions")?.size)
+        Assertions.assertEquals(2, update?.getList("actions")?.size)
```

Comment:
if you're changing this to 2, you should probably check the nested one is also of size 2